wsd/FileServer.cpp

```diff
@@ -2306,11 +2301,8 @@ void FileServerRequestHandler::preprocessIntegratorAdminFile(const HTTPRequest&
     csp.appendDirective("frame-src", "'self'");
     csp.appendDirective("frame-src", "blob:"); // Equivalent to unsafe-eval!
     csp.appendDirective("connect-src", "'self'");
-    // TODO: we have inline script tag and css in admin panel. can't have following CSPs
-    // csp.appendDirective("default-src", "'none'");
-    // csp.appendDirective("script-src", "'self'");
-    // csp.appendDirective("script-src", "'unsafe-eval'");
-    // csp.appendDirective("style-src", "'self'");
+    csp.appendDirective("script-src", "'self'");
+    csp.appendDirective("script-src", "'unsafe-eval'");
```

Comment:
if this was commented out before, do we really need to add it now?

Reply:
not really but I assume to safe to add these directive. before it was comment because we were using inline script tag in html and we removed so I added back the directivies.